server/src/controllers/order.controller.ts

```diff
@@ -54,39 +57,53 @@ const OrderController = {
     } catch (error) {
       return res.status(500).json({ message: error.message });
     }
+  },
+
+  async payAmount(req: Request, res: Response): Promise<Response> {
+    let isPaymentSucess = "1";
+    try {
+      request(
+        {
+          method: "POST",
+          url: `https://${process.env.key_id}:${process.env.key_secret}@api.razorpay.com/v1/payments/${req.params.paymentId}/capture`,
+          form: {
+            amount: req.body.total,
+            currency: "INR"
+          }
+        },
+        async (error: any, response: request.Response) => {
+          if (error) isPaymentSucess = "0";
+          else {
+            const userId = req.body.user._id;
+            const user = await User.findById(userId);
+            if (user != null && response.statusCode === 200) {
+              let i;
+              for (i = 0; i < req.body.orderId.length; i++) {
+                user.transactions.push({
+                  amount: 100,
```

Comment:
Since we have the total from `req.body.total`, we can use it in the value for amount.

Reply:
Sure, thanks. I shall rectify these by 8/04  11 am.